Tasks/BashV3/bash.ts

```diff
@@ -79,7 +79,7 @@ async function run() {
             if (old_source_behavior) {
                 contents = `. '${targetFilePath.replace(/'/g, "'\\''")}' ${input_arguments}`.trim();
             } else {
-                contents = `bash '${targetFilePath.replace(/'/g, "'\\''")}' ${input_arguments}`.trim();
+                contents = `${process.platform == 'win32' ? "" : "exec "}bash '${targetFilePath.replace(/'/g, "'\\''")}' ${input_arguments}`.trim();
```

Comment:
Nit: could you please move conditional logic out of template -  for some readability?

Reply:
It is not necessary changes so I just removed this condition.